.gitignore

```diff
@@ -14,7 +14,7 @@ node_modules/
 .env.development.local
 .env.test.local
 .env.production.local
-
+sample.env
```

Comment:
You wouldn't want to ignore the `sample.env` file. The reason it exists is to give any other developer an idea of what env variables to expect :)

Reply:
Applied